### PR TITLE
Refactored the app

### PR DESCRIPTION
**Pull request by siddhant1**

### CODE REVIEW

**Comment by siddhant1** [comment]
I have made changes according to the official typescript-node-starter by microsoft
Also I recommend adding the tslint.json to mention code-style rules

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 4. Make a copy of `.env.sample` file as `.env` and edit the configurations.
 
-5. Run `npm start` to start the server.
+5. Run `npm run build` `npm start` to start the server.
```
Comment (algomaster99) [inline README.md]: Running `npm run build` isn't required now. I have integrated the build process into `npm start` itself.

File: src/index.ts [hunk 2]
```diff
@@ -1,16 +1,15 @@
-const express = require('express')
-const dotenv = require('dotenv')
+require("dotenv").config();
+const express = require("express");
```
Comment (algomaster99) [inline src/index.ts]: Usually style guide prefer single quotes. That's why I'm following it.

Comment (algomaster99) [inline src/index.ts]: Usually style guide prefer single quotes. That's why I'm following it.

File: src/index.ts [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
 
-dotenv.config()
+const app = express();
```
Comment (algomaster99) [inline src/index.ts]: @domoritz I am keeping out semi-colons because it looks clean. What do you say?

File: tsconfig.json [hunk 4]
```diff
@@ -1,11 +1,16 @@
 {
-    "compilerOptions": {
-      "target": "es6",
-      "module": "es2015",
-      "lib": ["es2017", "dom"],
-      "moduleResolution": "node",
-      "sourceMap": true,
-      "esModuleInterop": true,
-      "outDir": "dist"
+  "compilerOptions": {
```
Comment (algomaster99) [inline tsconfig.json]: Okay, I will incorporate these changes. Thanks!

File: src/index.ts (hunk 2 above)
Comment (algomaster99) [reply src/index.ts]: It will be better to keep all imports in one place.

File: src/index.ts (hunk 3 above)
Comment (domoritz) [reply src/index.ts]: Yep. Prettier should add them automatically.

Comment (algomaster99) [reply src/index.ts]: @domoritz I have added a config which doesn't let eslint add them. Shall I remove it? I prefer no semi-colons though :sweat_smile: .

**Comment by algomaster99** [comment]
@siddhant1 Since #3 is merged, I am closing this :smile: .

**Pull request closed without merging**